GPU/GLES/ShaderManagerGLES.h

```diff
@@ -167,7 +167,8 @@ class ShaderManagerGLES : public ShaderManagerCommon {
 	std::vector<std::string> DebugGetShaderIDs(DebugShaderType type);
 	std::string DebugGetShaderString(std::string id, DebugShaderType type, DebugShaderStringType stringType);
 
-	void LoadAndPrecompile(const std::string &filename);
+	void Load(const std::string &filename);
+	bool ContinuePrecompile(float sliceTime = 1.0f / 60.0f);
```

Comment:
Maybe this should be slightly less, in case we would lose a frame to vsync?

-[Unknown]

Reply:
That will already be rounded very slightly down due to fp precision :)  But let's see how it works.